package.json

```diff
@@ -85,6 +85,7 @@
     "propose-safe-tx": "bunx tsx ./script/deploy/safe/propose-to-safe.ts",
     "postinstall": "patch-package",
     "remove-from-diamond": "bun script/tasks/cleanUpProdDiamond.ts",
+    "scan-selector-approvals": "bunx tsx ./script/migration/scan-selector-approvals.ts",
```

Comment:
same here, do we need this in package.json if its a one-time-use thing?

Reply:
good point, removed